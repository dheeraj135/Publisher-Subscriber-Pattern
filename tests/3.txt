U topic8 1
S topic2 2
T topic9 S1vFrKXjp8nDO4D
U topic8 2
S topic9 0
S topic0 0
U topic0 2
U topic3 0
U topic7 1
T topic0 zeCompbquChBDO1
T topic1 cFDqzauW6jdsiSM
T topic3 YETy8jPCnAPRrxh
S topic1 0
S topic3 0
U topic7 4
T topic4 0HAHJJaKOWHhsGK
U topic5 2
T topic4 UGK5UPcDR3BAbFg
S topic2 4
U topic9 2
T topic0 2Z2nRbXlktA0Czh
U topic0 3
U topic2 2
S topic0 3
T topic2 4RaACC1R0ZrrSRq
U topic1 3
U topic0 0
T topic8 DPNbAo7a0oI6ESs
S topic2 1
T topic0 Hbl1QuZEzj46MsS
U topic9 1
S topic1 2
T topic7 vWTGJsRFQJRNLXq
U topic2 2
T topic4 KRPPxfASg1OZ2yV
T topic4 qpitCo74Ej7OBCa
S topic1 4
U topic0 0
U topic6 2
T topic9 epAWf1bUA7XOBTC
T topic4 IAlvIBvoWrxkhd9
S topic3 4
T topic7 JmqFFakURHNEYpR
T topic2 ZP7a2orZQ1mdsvr
S topic5 1
U topic5 0
U topic6 4
U topic8 1
T topic5 1GKA0ElpAyLNxSj
U topic3 4
S topic4 1
U topic2 3
S topic3 1
U topic1 4
S topic1 0
T topic2 ZBWAUdG35knQGhy
U topic9 0
T topic7 IF0Q2Pon9b2CIia
S topic5 2
S topic5 1
U topic9 4
U topic9 4
S topic9 1
T topic8 OSZ67mCUrP0cVci
U topic7 4
T topic9 e0m0uAemfLXgTmk
T topic0 38yu1j0j9SyjvlP
S topic3 0
U topic2 4
T topic2 68Zy89t2LmL5Cmj
S topic7 3
S topic8 1
T topic1 Ztpiep0xGm7mqAj
T topic9 9Ud7GOV6OIJKu0J
T topic1 kCZYAodLtjGShub
U topic2 1
U topic0 2
U topic0 1
U topic1 2
T topic0 t9rAc3Qd45OT7hj
T topic8 yEjB0s1PttdH0ik
T topic9 VjqLncjJJWTS5p8
T topic5 5BqNl7MJ8Ddn7Ct
U topic1 3
T topic8 dWAIeWMI993ekgx
T topic5 tPbCK9c3o0eTQzf
U topic0 4
U topic9 1
U topic8 4
T topic2 viBXOQj7xsPePTW
U topic7 4
S topic9 3
U topic4 2
U topic7 4
U topic4 1
T topic9 JHXXpmH9jhPNK42
T topic0 ShnOiYIb3Uqs9xu
S topic0 4
S topic0 4
T topic5 6aseH8zqMydcBHD
T topic8 XeV0QBP3G3t44zk
T topic9 llrLEQlgKf2TWcS
T topic4 G6yc2iCOiOhrbE7
S topic6 0
S topic4 0
S topic4 3
U topic5 4
S topic9 2
T topic4 eI5c9uPca34yKTV
U topic9 2
S topic6 4
U topic4 3
S topic4 2
T topic5 DEtDp0J69FNzkix
T topic7 sSnQKeEdIjjYu19
U topic0 1
S topic2 1
T topic1 kZQek29abUFiKWa
U topic9 1
U topic5 1
S topic6 3
S topic3 1
S topic2 0
S topic8 4
T topic6 ZXHSsRSblYb5bpZ
T topic9 lG1GB5Q1bZdeaQK
U topic2 3
S topic6 2
S topic2 2
S topic6 1
S topic3 2
S topic0 0
T topic0 Cl0JRYRXuHEzr3b
S topic4 2
S topic1 3
S topic3 1
T topic7 ZUl79LFGhQQhhD3
T topic6 FCflXOVAvgAZfdz
S topic8 2
T topic4 xAaEtoM4HRXetgR
T topic5 DB1xVKI2P6MXtNF
S topic5 0
U topic7 2
S topic6 1
S topic0 4
T topic7 BbQMgsvhBtkiTH9
T topic8 F5d2MDsvXy0LzqS
S topic7 0
U topic1 0
T topic9 pXYxTDqKyx3gyeO
S topic4 3
T topic4 QkbvnGxIjDQnrtI
S topic7 1
S topic4 0
S topic8 4
U topic1 2
T topic7 1LcQKBYi7g6bADj
T topic3 vm8R5k1iTfvS1rL
T topic8 cALQ7U9FO0GTIf5
U topic0 3
U topic2 4
U topic0 2
T topic3 tqKrTlaAYT2JZTL
S topic0 1
U topic6 4
T topic4 GDkYo3KagYSjfry
U topic2 4
T topic1 rRtBU2nBTRqzzY0
S topic3 3
S topic8 3
S topic3 0
T topic4 CvpxxbPpiRaPWBS
S topic0 4
T topic9 TplukT02mmK8yGr
S topic5 3
T topic7 61Mz6AFru58qrHP
T topic0 BWDUHemh7tMtWOf
S topic0 4
T topic7 ivnBK1aJ0xqIkPV
U topic3 4
U topic8 0
U topic6 4
U topic6 4
S topic6 3
U topic4 4
T topic4 LyuHcX1uhxG2B2j
U topic1 2
T topic1 8oWPzJC1cbPz5YC
S topic5 4
U topic5 3
U topic1 0
U topic7 3
U topic4 0
S topic3 4
T topic3 2FDx1fdrsDchQDb
U topic4 4
U topic8 1
U topic6 4
T topic5 TsK7mCllBem52fQ
S topic5 3
T topic0 ssT6kcUCmdQ10un
U topic7 3
S topic7 4
S topic0 0
U topic7 4
T topic6 pCxaxnRhb4tv3pe
U topic9 1
S topic4 3
S topic2 3
U topic0 1
T topic4 arIA0ExPy7Wi4iQ
U topic7 3
T topic9 YCXqZ0MVDJdeBUM
S topic0 4
U topic8 3
T topic3 9VnUuqae1sY3miE
T topic3 WShBJBcirhsUA6N
S topic1 0
U topic8 2
T topic7 SnJ0oxugNvrl1y9
S topic1 2
T topic1 aAZqQLHFshObHDS
T topic5 sQJqFucd29JTirL
T topic3 F18q9ehylgdpjci
T topic9 DZk0mOuBF5bqg47
S topic8 3
T topic7 aaMhVG3W6oSM79E
U topic8 1
T topic1 usU1XQdLYNbe9ls
S topic0 0
U topic8 1
T topic9 N2y6huchMWu5zhe
S topic5 4
U topic2 4
S topic6 3
U topic1 3
S topic6 2
U topic1 2
U topic1 2
S topic8 4
U topic7 3
U topic5 2
U topic1 4
S topic5 3
U topic3 4
S topic5 4
U topic2 3
S topic4 1
T topic8 K425cz7hvVotvQI
S topic8 2
T topic8 VBPpqrngjyErsaz
T topic6 OCt6TSUeE2GOpDV
U topic0 0
U topic3 1
U topic2 0
S topic4 2
U topic3 3
T topic5 ubb05V6BgzbegHb
U topic7 0
T topic8 ILH21gRo2n866sP
U topic5 2
U topic3 1
T topic6 OxIqr6eI31tTARQ
S topic9 2
T topic5 CFj1n34fhzC90D4
S topic5 3
S topic4 3
S topic1 4
T topic3 kMmYL91ElS0nz7y
S topic4 0
T topic2 gDtvlAhdiffg8HK
U topic6 0
S topic5 3
S topic7 0
U topic8 4
U topic3 3
U topic7 2
T topic3 Z0uf1eOz7gRhjzT
S topic5 4
S topic2 1
U topic6 4
S topic2 2
U topic8 0
U topic8 3
S topic4 1
S topic2 4
S topic4 3
S topic4 2
S topic3 2
U topic6 2
S topic1 3
S topic6 3
U topic2 4
U topic4 1
S topic1 2
T topic6 e7zQ4brO3FvxQou
T topic2 Fo6UZEvO86XdQpo
S topic6 0
U topic5 2
S topic9 0
U topic1 2
U topic9 4
U topic5 1
U topic6 2
T topic4 bRyi6yUdh4ZOZiP
S topic3 3
T topic8 UY1m35yKB2MCbvr
S topic9 4
S topic8 4
S topic8 3
U topic4 0
U topic9 2
S topic8 4
T topic2 KCed4cU5PVyYMrb
U topic8 4
S topic1 0
S topic8 0
S topic9 0
T topic0 Rriz0qVarRlKgAx
T topic4 2E6h6L8AbxJf9Om
U topic3 0
U topic0 0
U topic4 0
S topic7 3
S topic9 2
U topic3 0
S topic9 1
T topic8 XDQzjtnYNcKfB5N
T topic9 LFebt1C8iufsGPP
U topic2 2
U topic2 4
U topic1 3
U topic9 3
T topic8 dgQ4T0O5YtceCQF
U topic5 4
U topic6 1
T topic5 LrZoMOWeB0sjz39
S topic2 4
U topic0 4
T topic9 qK8UyTucXQnnDZ2
S topic9 2
T topic0 4neQf0EMWZupiR4
S topic2 3
U topic1 1
U topic6 1
S topic9 0
U topic1 4
T topic5 mCvE1qnJ6YIF2Zm
S topic9 0
T topic7 NK7kvR2YnvL1ltQ
T topic5 VNNg7XgxUpnIptp
S topic8 0
S topic2 2
U topic6 2
S topic6 2
S topic8 0
T topic2 MDQhghSNRLEDb2V
T topic8 aoVKXbZoRLk8WGM
U topic4 0
U topic8 3
T topic3 1xVNFAD18D73XbD
S topic2 0
T topic6 hbvFVSGtUHoUqM5
S topic3 4
S topic8 0
T topic6 E1WL37Nq1dUCc9N
U topic0 3
U topic9 0
T topic2 3ck0WbLzgZhufLo
U topic7 0
S topic5 4
T topic0 F1Kl0LcsOzaPg3J
U topic7 3
T topic6 zLKi81WTC56U8LY
S topic1 3
S topic3 3
U topic3 3
U topic8 1
S topic9 2
T topic5 Xfm2yAMOborH5nC
S topic6 4
U topic1 1
S topic9 4
T topic4 reTckjnSV2AAn05
S topic8 3
U topic2 2
S topic7 4
S topic2 1
S topic0 4
T topic7 91iiVgu3ne1BbYN
T topic1 VPJR4YGBVEkmJZ7
S topic1 2
T topic1 MLPJ9TfrqVvdKkR
S topic8 1
S topic7 3
U topic7 1
U topic5 3
T topic4 FbRzSBFpVFIy4rg
S topic7 2
S topic2 1
T topic5 yx2qoq6uK4qqQLm
U topic2 2
T topic0 nF1fg0IqzIdKMO6
S topic8 0
T topic2 j4fuNPeX0UcE2Ht
T topic3 Yb4pupkzZ3TkEuu
U topic2 1
U topic9 1
S topic1 2
T topic6 TipeNm8jUh5OZpu
U topic4 0
U topic4 2
T topic9 zRBjAW6ia9GE5Sv
U topic8 1
T topic5 VhWEGcAYEcYxsyq
T topic8 7qSLGlCP6CItAUm
S topic5 0
T topic2 5NGerzWS9YUoY7N
T topic9 6v6CMCEv8pTg1G5
S topic7 2
U topic3 2
S topic1 1
U topic0 0
T topic4 rk7Oxfczz5FkjfC
U topic3 1
U topic3 0
U topic0 0
U topic2 1
T topic9 dgqGIdv5AOzXdQL
S topic8 0
T topic8 dhlU2g3mr6kkR84
T topic6 FlpMV4Q1gSDzY7v
T topic9 hVYiWoGuFekLBJ1
U topic3 0
S topic5 1
T topic0 Dxr6HobWN6oLQgL
U topic3 0
U topic0 4
U topic3 2
T topic3 JDKQCMVYAPQYCzF
U topic0 1
T topic3 arsJYeD27eoHNW9
S topic2 1
T topic1 qsFIOmgMHijpycx
S topic9 4
S topic0 1
T topic4 EoQXia2NNs7ymdE
T topic9 yC1R6jUrkQUo394
T topic7 o56bWIvST3sqCfQ
S topic1 4
U topic9 4
U topic9 0
T topic2 LguXQg1XZPCeiPe
U topic5 2
S topic9 2
S topic7 1
S topic1 4
U topic5 0
T topic9 USjpdYyokFJfBaQ
U topic0 3
S topic2 2
S topic9 3
S topic4 1
T topic4 Wiyoyfl7TBoZKDK
T topic0 0J1XWqFJVv1IcxZ
T topic1 2U0BUuB0HRBG6uJ
T topic4 MDYZZerSj2WUrlk
U topic2 0
T topic1 fGAXCoivcnTWKf2
U topic9 0
U topic1 4
U topic2 0
S topic7 3
U topic3 3
T topic9 lSKocFukvo03LLV
U topic3 1
S topic9 4
S topic0 2
T topic5 7CrKBQRGLLtIqe0
T topic2 mTTU4mMlzxT2IzP
U topic0 0
T topic6 uc0nB8HI5Do94c2
U topic6 4
U topic8 1
U topic1 1
U topic0 0
S topic8 0
S topic8 0
S topic6 2
U topic4 4
S topic0 2
T topic8 AYLSBiqOLB52cUV
U topic0 1
S topic2 0
T topic4 m3iI3qCYVdDMlxy
T topic3 2zB1K2vYshTevq1
S topic8 3
S topic6 2
T topic4 8g1kza7R5SKEHfO
S topic0 4
T topic7 sRuoNzoAC4KfpRQ
S topic4 1
T topic7 m5ZoD8QmVbNLfts
U topic0 3
U topic8 1
S topic6 0
S topic9 0
S topic8 0
T topic3 tq4fEUUvWzy9cOf
T topic9 EREYo8YAP8mLpGB
U topic1 0
U topic9 0
U topic0 0
S topic6 4
U topic0 0
U topic1 2
T topic8 dAXXfeSbtZpPKyD
U topic5 3
S topic0 4
S topic1 1
U topic3 0
U topic0 3
T topic0 PrkznLANZBNH2jA
S topic7 1
T topic4 8JjJoY7X67HiBas
U topic8 0
S topic8 2
T topic5 XygJxXte0Rim9b1
U topic1 2
U topic2 2
S topic0 1
T topic0 R2pgxA3aEKTCo89
S topic1 1
T topic4 LtmcLtgWp2zTd2g
S topic0 3
U topic5 4
T topic0 pXNgjMnWjvAU04z
S topic6 2
S topic2 1
T topic1 54MH9WoOJD9lYzK
S topic5 4
S topic8 4
U topic1 4
S topic9 1
S topic2 1
U topic1 2
U topic8 4
U topic0 1
T topic2 KAVjf77bX8A3GOc
T topic2 eJggk9nLS1gKqCy
U topic7 3
S topic3 4
T topic8 a4IUgXNRLuTGkA6
U topic6 4
S topic5 4
T topic4 79bXmAKiB8JdNld
S topic0 1
S topic1 4
S topic9 1
T topic5 A17cM6zT7pgIJax
S topic4 1
T topic6 U4uLxOPQMst4Zb9
S topic3 3
S topic6 3
U topic0 4
S topic3 2
U topic4 1
S topic1 0
T topic3 SqHRVL6PgomSq4k
S topic7 1
T topic1 r66PgFYKklrCbpX
T topic0 B1kYyyge7GacidP
U topic1 3
S topic8 4
U topic2 0
T topic4 sK0W9MyW5IsxUhy
U topic4 1
U topic5 1
T topic3 sjjSHTIpohIvL84
T topic1 jSa6CSOWzajWLNB
U topic9 2
T topic8 TMIp8gKUQtNoI3I
T topic8 SZbrM2h5HFbGc0Q
U topic0 3
T topic4 KGbkYSEpZ7UFYRA
S topic3 0
S topic4 2
U topic5 4
S topic5 1
S topic4 4
U topic4 1
T topic4 JGYGvpo6lIEqKZW
T topic0 gXCGZ3IniTS1McG
T topic1 JoKkbokF5AUElc6
U topic9 0
S topic4 1
T topic7 F9kCQrcryRXKhI1
U topic2 1
T topic4 ogl2qDYI6gDvsF9
T topic5 GWdPWB2i1p7CnFe
U topic2 4
U topic3 2
S topic6 4
S topic7 0
S topic0 1
T topic4 kYth0FRnnlTXmUR
U topic2 1
T topic9 Of6jtG4easq786f
U topic7 1
T topic0 DTTZOnHlF2XgTLu
T topic3 lmTV8vXzdJUHYZI
S topic3 1
U topic9 1
S topic8 4
T topic6 EeCKa2jrla6Ha3d
T topic2 ZtI9sg04Kl10kfH
T topic0 fUtpD80ueYdisoR
T topic6 xNqJaVPXZ5pO5AR
U topic4 1
T topic7 2opzdU1UIcWGsyA
T topic3 F5CILCyMno2OjZ2
U topic3 0
T topic8 Rj2benfNYsSKcr7
U topic4 2
U topic5 2
S topic1 2
S topic7 2
U topic7 2
S topic5 4
T topic5 XMi9tFfAJpi5Mm5
S topic3 4
S topic4 1
T topic5 FVxtV5NbKghWg1B
T topic0 l8UOpTQIIHvxjbB
S topic0 4
S topic0 4
T topic3 Yn6IkC59SAxYrFa
U topic6 0
U topic5 4
S topic2 0
U topic7 0
S topic2 3
U topic0 3
S topic1 1
U topic9 0
S topic4 4
S topic8 4
U topic0 1
U topic4 0
T topic9 2d8SdlHeChGfBpO
U topic0 1
T topic7 o74U5aZtHFCFDIQ
S topic0 1
T topic3 VMYzMCEuY0MqTNQ
T topic1 Yc6T0FdYrpbzkbG
S topic2 1
S topic7 3
T topic2 jjVVsMpBzFZTsgU
S topic4 1
U topic7 1
T topic7 gQJvDiOqjYiOViF
U topic2 4
U topic8 4
U topic8 4
S topic0 4
T topic9 OMOZZ890p0YBqRo
T topic9 O9uDkgXIdzkQBJ3
U topic9 3
S topic1 0
T topic2 IYIEKe3OybRtENe
U topic0 1
S topic8 1
U topic1 2
S topic5 3
S topic5 3
S topic4 1
T topic2 BnT0UIcxxX3XfPG
U topic4 4
S topic6 2
U topic5 3
T topic3 RLlUmTUXqHXT2HE
T topic6 45mjsinnogmYIin
S topic1 0
T topic6 XvBFUjExe8GDUxq
S topic7 1
S topic1 2
T topic3 dYng2831BdMZmKZ
U topic0 2
U topic6 1
T topic5 FSf5ApZm6VTEVzD
U topic0 1
T topic3 RPJBVBkmAclQqy2
T topic7 sd6um7cI7VANFbi
U topic8 0
S topic9 3
T topic3 JUyDlAsQqzdblUg
U topic8 2
T topic9 feYrPEq7FrV80xH
S topic2 4
S topic1 3
T topic5 ED73dBgCZ8MiOZ5
T topic2 r2fjr6jz5s13Fjr
T topic2 PhBujjxDIEsQ0ee
U topic1 2
T topic8 46ml4vhMWVSYGhX
T topic9 7rp55mkkKWYWXzo
T topic1 rdcpa9vtkeCxMA1
S topic8 2
S topic9 0
U topic5 3
U topic6 1
S topic4 3
U topic1 4
U topic0 0
T topic2 2OSBevD6YYfpSaZ
U topic7 3
U topic9 3
S topic1 3
U topic3 3
S topic2 0
U topic7 2
U topic1 4
U topic8 1
T topic0 oj7iIc2oIfhNSm5
S topic7 2
T topic5 igAM8jJizKcqFjm
T topic5 Z9y2l7WL2SvckQ6
U topic6 0
S topic5 1
S topic7 3
U topic0 3
U topic6 0
S topic6 0
T topic6 QrsLWtIXmusmZqR
T topic1 1rES2KfRPKJr4Ab
S topic1 1
T topic4 F2jdafCqE8xB2oa
U topic3 2
S topic8 0
U topic5 4
S topic3 0
T topic3 TxihZzNWo30xQoI
U topic7 1
T topic7 X8uxR3XDoUFfAR4
T topic3 2NCNBBHWTQmvS0n
T topic0 6X9yPxfISOGcLk9
U topic6 4
T topic0 2K1xq9HiDE8hAD6
U topic3 2
T topic3 uPVZlvYngaBULc6
U topic6 4
T topic0 6MgaAosGBoH2MKx
T topic5 SjTGn8sOSh9XXDA
S topic7 1
S topic9 0
T topic5 37iE77FOFrF5HjG
T topic9 z1uYdgzOG295rGB
U topic1 1
S topic0 1
T topic9 Xz9UcJ86mJDdYdy
S topic3 1
S topic8 3
U topic7 2
U topic2 4
U topic3 1
U topic8 4
S topic6 1
S topic9 4
U topic7 1
S topic2 0
T topic8 RpCRAGgIIxn4sfV
T topic2 X14UqPy6lEKbFoW
S topic7 0
U topic2 4
S topic7 2
T topic3 QLN0XIcuvcdlbIY
S topic3 0
T topic5 9VnIrTNltvhMJ35
U topic4 3
S topic0 4
S topic2 0
U topic2 0
T topic6 afbaSz094MoBJmo
S topic1 0
U topic3 3
S topic7 3
T topic7 EfxOZHlARgWbZQk
S topic3 1
U topic5 2
T topic1 VKYWHPvlK86XK3y
S topic0 4
U topic5 3
T topic4 FyBhjKHaoL1YdVz
U topic9 3
U topic4 1
S topic0 4
U topic8 4
U topic9 3
U topic4 2
U topic8 2
T topic6 xbrP9Mxhe8hvBQo
S topic0 0
S topic3 1
U topic4 1
S topic4 0
T topic1 bijcMNzKevC6C7L
U topic7 1
T topic5 pH7dJYXW6Z6FJXj
T topic5 4hgORevMKTexpAn
T topic9 xgQA2C7uCP22Hr4
S topic8 0
S topic8 0
S topic4 2
T topic5 25IyKjJYgneStQb
T topic7 7keXF6E6bXUeh59
S topic8 2
T topic6 RFDxhZ0OeYHT5IA
S topic6 3
T topic2 FOzCUaHefmT1KJs
T topic4 CvjAs3UMtIEM943
U topic6 1
T topic1 PKJFKMtbuCnMap6
S topic6 4
T topic5 hAkhUB52NsNNs6n
S topic2 3
S topic9 3
U topic9 3
U topic5 3
T topic0 INgRzL2kZiQuioy
T topic9 jj6RWka4RtS3mh6
S topic5 3
S topic9 0
T topic1 5msmdTU7WnJ5DUP
U topic7 2
T topic8 ceSmQU2W3kpyOA9
S topic3 1
T topic3 jH0zT4OootZIA3V
T topic9 0lvfXGW5l7hdcL6
U topic8 4
S topic4 4
T topic7 KGBc4lKn2oBcZWG
T topic3 niK4M1h0c9biTPq
T topic6 6ERheLxBye7qdnl
U topic3 0
U topic9 0
U topic7 2
S topic5 1
S topic7 1
T topic3 6h0oJzsZICN993X
T topic1 B7Ch4GYh3KqtZVv
U topic2 1
T topic8 lRiSZJsAnGLKex8
U topic7 0
U topic5 2
S topic7 1
S topic8 3
U topic6 1
T topic8 gDzs2SQIxjjQ9J0
T topic5 I3dtKv2ND05UnfW
U topic2 3
S topic2 2
S topic7 2
U topic7 0
U topic0 3
S topic2 0
T topic6 CrqxYtVqxjj6U3p
U topic0 4
S topic6 2
T topic7 f0WFUDZuhoMdDf7
T topic3 AF5ig8ZSOzvVgZk
S topic4 3
S topic4 1
T topic4 kskA9Gcxmetehvu
S topic3 4
U topic2 3
T topic4 NiWv1BAGO6J9eP3
U topic0 0
S topic7 3
U topic2 3
U topic7 0
T topic4 YsiPIyvlhUQl9Em
S topic0 3
S topic9 3
U topic4 0
S topic0 3
S topic6 1
S topic5 4
U topic5 3
S topic9 4
T topic4 1V6EK5YKf9VNZjK
T topic7 4sz8srlMyCgSS1k
T topic0 MCCfr3fJk1KyEl5
T topic4 Emo2TXEUq6Q1aaJ
S topic8 4
U topic5 3
T topic8 HdTveJiFyg0SyrV
T topic1 N5xEA14qC6FIbZ4
S topic2 3
S topic1 1
U topic8 2
T topic6 vXSmSsqs49LTgN4
S topic6 3
S topic1 4
U topic8 1
U topic1 2
T topic0 PqSYYzTsLk3WJxV
U topic5 1
T topic2 czg8gXaO8D3gmdW
U topic5 2
T topic8 6LZLsxt0RD1jMt1
T topic5 6uqkqQ9jJkEZIF2
T topic7 JKinY3accmnIsL1
S topic8 3
S topic7 2
U topic9 1
U topic8 4
U topic7 0
U topic6 2
S topic6 1
T topic0 bKuin6KT3RrNXy6
U topic1 4
U topic9 3
U topic7 1
U topic0 0
U topic3 4
U topic7 2
T topic3 VslXsH2L9tuFSFx
U topic8 3
U topic0 3
T topic2 Qhu5EK0O6Ru0HWh
S topic3 4
U topic1 3
T topic7 AESzogVboGjjvAU
S topic0 4
T topic5 m8X7IKQDV2hMgD7
U topic7 3
T topic9 AOR58WF0jhmfzSa
T topic3 ZT4sipUFsE8CN8H
S topic0 3
S topic2 1
S topic7 4
S topic3 3
S topic2 0
T topic7 vKYK3LVyNoGzx7z
U topic9 3
S topic9 3
T topic8 xmE81aGKm52k0y7
T topic4 9hgjyBy26ETiYro
U topic7 3
U topic2 2
S topic6 4
S topic9 2
U topic6 0
S topic1 0
T topic7 Vsapb5lCG58Nt3s
U topic2 3
T topic5 d3fDkQHEn2vVv4P
U topic6 0
U topic0 1
S topic0 1
U topic5 1
U topic2 1
T topic1 14k4BJbUfu1IIyB
S topic6 3
T topic9 mtLqJtipZQrHq3E
S topic7 4
U topic2 0
U topic5 1
T topic1 Eo4ljfnJMbmN6rg
U topic1 4
U topic5 2
S topic8 4
T topic1 YQBLK2S9vjAJXSu
S topic8 3
T topic7 ec2Eu2JtK7e1sIW
S topic2 0
U topic8 4
U topic4 0
U topic1 1
S topic1 0
T topic9 YPooIM2L59SjMKf
T topic7 YGdBRMPo7KUfZcj
T topic5 8aBTfklS11r2Si1
T topic8 MFyCtBiX82GCxRM
T topic9 OQayagxcaR0opuX
T topic6 0YVOl7Dzcg6QFHu
U topic2 0
S topic8 1
T topic9 Z8XA4oe3sKsCPFQ
T topic9 N4aXv2z37ItSklQ
T topic8 8Qb5PgeCq1GXW6P
S topic7 1
S topic0 4
T topic0 OantPzqagj9NLAY
S topic8 4
S topic4 0
U topic1 0
U topic4 3
U topic0 1
U topic1 2
