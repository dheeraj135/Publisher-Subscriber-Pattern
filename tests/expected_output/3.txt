0oG0dsCPYCrMaUA
evV9a93eaGhe27s
XOudc17RgxpUqBF
pmeTrCr1EHtkORM
cRPrxCXeAWngK9n
EjUaxWo0SCev67l
ilTzJc0fjys3WVv
nFbBYzLYcryHMzW
2jMlJFEhZIR0JDt
IsCSlBKvLYvBtej
3gU7I5khQHcjc38
UqdvKGAHocTCUfr
ena2xGm79CIHiq6
0D15Sm8aycBVXFm
ALWTYaMIeijd9rp
